photo-points/Services/AdminReviewServices.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace photo_points.Models
+{
+    public class AdminReviewServices
+    {
+        private IAdminReviewRepository _capturerepo;
+
+        public AdminReviewServices(IAdminReviewRepository capturerepo)
+        {
+            _capturerepo = capturerepo;
+        }
+
+        public IQueryable<Capture> GetAllCaptures()
+        {
+            return _capturerepo.GetCaptures();
+        }
```

Comment:
Also include a service that returns all of the captures not yet approved as an IEnumerable<Capture>

Reply:
Should be added now in my latest commit